apiary-metastore-listener/README.md

```diff
@@ -118,13 +143,18 @@ The following shows an example JSON message representing an "ALTER_PARTITION" ev
       "eventType": "ALTER_PARTITION",
       "dbName": "some_db",
       "tableName": "some_table",
+      "tableLocation": "s3://table_location",
+      "tableParameters": {
+       },
       "partitionKeys": {
           "column_1": "string",
           "column_2": "int",
           "column_3": "string"
        },
       "partitionValues": ["value_3","2000","value_4"],
-      "oldPartitionValues": ["value_1","1000","value_2"]
+      "partitionLocation": "s3://table_location/partition"
+      "oldPartitionValues": ["value_1","1000","value_2"],
+      "oldPartitionLocation": "s3://table_location/old_partition"
     }
 
 # Legal
```

Comment:
Update below to say "Copyright 2019, Expedia Inc."

Reply:
done.